resource/advanced/alter.md

```diff
@@ -20,7 +20,7 @@ def alter(self, resource: str, class_name: str, traits: dict) -> Union[dict, byt
 &nbsp;
 
 {: .experimental }
-> _Only a subset of available **Segments** and **Traits** are considered **Stable**. See [Segments](../segments_traits_operators#segmetns) and [Traits](../segments_traits_operators#traits) for more details._
+> _Only a subset of available **Segments** and **Traits** are considered **Stable**. See [Segments](../advanced/segments_traits_operators#segmetns) and [Traits](../advanced/segments_traits_operators#traits) for more details._
```

Comment:
broken link.

Reply:
Change `#segmetns` to `#segments` in the url to segments documentation.

Reply:
that typo was in 3 places. All fixed